### ISSUE TITLE
[BUG] Setting color_system seems to affect more than just color

### ISSUE THREAD

**Issue by ronf** [willmcgugan/rich#882]
**Describe the bug**
I was experimenting with setting force_terminal and color_system on a Console object and got some unexpected results. I was expecting that setting force_terminal to False would suppress all rich text styling and color and that setting force_terminal to True and color_system to False would only disable color, but not disable things like bold or italics.

Instead, what I got was that setting color_system to None seems to disable all color _and_ styling, and setting color_system to anything but None (such as 'standard' or '256') seems to enable both color and styling, regardless of what force_terminal is set to.

**To Reproduce**
I was expecting the following output:

```python
from rich.console import Console
c = Console(force_terminal=True, color_system=None)
c.print('[red bold]Expecting bold but not red[/]')

c = Console(force_terminal=False, color_system='standard')
c.print('[red bold]Expecting neither red nor bold[/]')
```

What I actually get is neither red nor bold for the first case (disabling more than just color), and both red and bold for the second case (outputting both color and styling even though there's no terminal):

![Screen Shot 2021-01-05 at 8 29 38 PM](https://user-images.githubusercontent.com/1901085/103729212-bed02c00-4f94-11eb-9451-58f30819d28f.png)

The first set of output here is neither red nor bold, and the second set is both red & bold.

Setting force_terminal to both False and True works ok with color_system as 'auto'. However, once color_system is set explicitly, it seems like it overrides the setting of force_terminal, either enabling or disabling both color & styling, rather than only affecting color.

**Platform**
Rich 9.5.1 on macOS Mojave (10.14.6) with Apple's Terminal.app.

**Comment by willmcgugan** [comment]
This is the expected behaviour.

The `color_system` attribute is the only one which directly influences ansi codes related to color and style. Setting it to `None` disables all ansi codes, so you get plain text. It is possibly misnamed, but `color_and_style_system` felt too verbose. 

When `color_system` is `auto` Rich will look at the `is_terminal` attribute to decide if it is writing to a terminal or not. If not, it will set `color_system` to `None` to disable ansi codes. You can override the auto-detection with `force_terminal`. You may want to do this if terminal auto-detection fails, or you want to write to a file but you still want to generate ansi codes.

`is_terminal` also affects a few other more exotic ansi codes, moving the cursor, bell, etc.

**Comment by ronf** [comment]
Thanks for the explanation. In that case, I have a slightly different question. What arguments do I pass to Console() to get styles like bold and italic to be enabled, along with other things like moving the cursor and the bell,  but with color completely disabled? I'm thinking some of my users might want styling but not colored text, even if their terminal is capable of it.

The reason I'm setting color_system and force_terminal directly rather than relying on auto-detect is that my use case has multiple remote terminals connected via sockets to a shared server process, so I can't rely on the TTY associated with that process to set terminal type, terminal size, or capabilities like color. That's not really a problem, though, as I'm able to create a separate Console() object for each remote socket, and set the terminal size individually for each.

The problem seems to be that there's no value for color_system that lets me keep styling enabled but disable color. I thought there was using "auto", but even when I run directly on a local terminal I see now that setting a terminal type to something like "xterm" which doesn't imply color support leaves color_system set to "standard" and color output enabled. The only thing "auto" seems to support related to this is the ability to turn off everything when it thinks you are on a dumb terminal.

Would you consider adding a color_system value like "no_color"? That could be used as part of "auto" when the environment variable NO_COLOR is set (which is something you already check for, but today it disables more than just color), and I think it would support my use case. I'd be looking for it to allow all ANSI escape sequences _except_ for those that change the foreground or background text color.

**Comment by willmcgugan** [comment]
> Would you consider adding a color_system value like "no_color"?

I would have thought curmudgeons that don't like color would also dislike bold and italic.

Yes, I will consider adding it. I'd probably call it "monochrome". No promises on the time scale...

**Comment by ronf** [comment]
Thanks - "monochrome" sounds fine to me.

In the meantime, I'll probably offer only a switch to enable/disable all rich text (both styles and colors) as you suggested, which I should be able to do by setting `color_system` to `None`. I'll still need Unicode output and things like table layouts to work, but it looks like those are still fully function with that set.

**Comment by willmcgugan** [comment]
Slight change of plans. I've added a `no_color` flag which will strip color from the final render. This is independent of the color system.

See https://github.com/willmcgugan/rich/pull/901

**Comment by willmcgugan** [comment]
Fixed in v9.7.0

**Comment by ronf** [comment]
Thanks very much - I've upgraded to the latest version and this new functionality is working great!
